en/docs/integrations/integration-concepts.md

```diff
@@ -1,57 +1,54 @@
- - **Manual**: To run and test an integration manually.
- - **Calendar**: To trigger an integration based on events in a Google Calendar.
- - **GitHub**: To trigger an integration based on GitHub events and actions.
- - **Salesforce**: To trigger an integration based on topics in a Salesforce account. 
+ - **Schedule**: Triggers execution according to a given schedule.
+ - **Manual**: Triggers execution manually by clicking 'Run & Test'.
+ - **Calendar**: Triggers execution based on events in a Google Calendar.
+ - **GitHub**: Triggers execution based on GitHub events and actions.
+ - **Salesforce**: Triggers execution based on Salesforce events. 
 
 ## API Call
-An API call in Choreo integrations allows you to connect to and interact with an external service through your integration. You can use an API call to interact with generic protocol-based APIs and application-specific APIs.
+An API call in Choreo allows you to connect to an external service through your Choreo integration. You can use an API call to interact with generic protocol-based APIs and application-specific APIs.
 
 Following are the connections you can use to interact with generic protocol-based APIs:
   
- - **HTTP**: To interact with the HTTP API and communicate with an external endpoint via the HTTP protocol.   
- - **SMTP**: To send emails via the SMTP protocol.   
- - **POP3**: To receive emails from a POP3 email server.  
- - **IMAP**: To receive emails from any email server that supports the IMAP protocol. 
+ - **HTTP**: Communicates with external endpoints over HTTP.   
+ - **SMTP**: Performs email related operations over SMTP. 
+ - **POP3**: Retrieves emails over POP3. 
+ - **IMAP**: Retrieves emails from a mail server over a TCP/IP connection.
 
 Following are the connections you can use to interact with application-specific APIs:
 
- - **Twilio**: To interact with the Twilio API and perform operations such as sending messages, WhatsApp messages, and making voice calls.
- - **Slack**: To interact with the Slack API and perform required operations. 
- - **GitHub**: To interact with the GitHub API and  perform required operations. 
- - **Gmail**: To interact with the Gmail API and perform operations such as creating, modifying, and sending emails.
- - **Google Calendar**: To interact with the Google Calendar API and perform required operations.
- - **Google Sheets**: To interact with the Google Sheets API and perform operations such as creating and listing spreadsheets.
- - **Salesforce**: To interact with the Salesforce API and perform required operations. 
- - **NetSuite**: To interact with the NetSuite API and perform required operations. 
+ - **Twilio**: Connects your service to Twilio API to perform operations such as sending messages, WhatsApp messages, and making voice calls.
+ - **Slack**: Connects your service to Slack API to perform operations in Slack, such as sending a message to a channel, deleting a message, etc.
```

Comment:
Why do we say `service` here instead of `integration`? Check all the places.

Reply:
Isn't it that you are actually connecting your service to an external system in order to create an integration?

Reply:
No. We don't needt to have a service to create an `integration` in choreo.

Reply:
Fixed.